.github/workflows/test.yml

```diff
@@ -5,13 +5,8 @@ on:
   push:
     branches:
       - master
-      - packagecloud
-      - 'release-*'
-    tags-ignore:
-      - 'debian/*'
```

Comment:
maybe not relevant for this PR but this will need a "tags: ['*']" to be run when a new release tag is created.